paper-icon-button.html

```diff
@@ -135,6 +139,21 @@
        */
       icon: {
         type: String
+      },
+
+      /**
+       * Specifies the alternate text for the button, for accessibility.
+       */
+      alt: {
+        type: String,
+        observer: "_altChanged"
+      }
+    },
+
+    _altChanged: function() {
+      var label = this.getAttribute('aria-label');
+      if (!label) {
```

Comment:
:white_check_mark: Fixed! I just added a check to update the `alt` and not stomp over the user-entered value.